website/docs/components/button/partials/code/how-to-use.md

```diff
@@ -1,30 +1,32 @@
+## How to use this component
+
 The button component is used to trigger an action or event. For accessibility, buttons should not be used to route to a URL.
```

Comment:
I recommend moving this to `overview.md` which will put it at the top of the first tab as it's good general guidance. 

I also recommend adding more details or resources around **why** buttons shouldn't be used for URLs instead of relying on "for accessibility". Feels like a good opportunity for education on the topic.